src/Network/Network/NetworkManager/DeployAzNetworkManagerCommit.cs

```diff
@@ -21,12 +21,17 @@
 using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
 using Microsoft.Azure.Commands.Network.Models.NetworkManager;
 using System.Linq;
+using Microsoft.WindowsAzure.Commands.Common.CustomAttributes;
 
 namespace Microsoft.Azure.Commands.Network
 {
     [Cmdlet("Deploy", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "NetworkManagerCommit", SupportsShouldProcess = true), OutputType(typeof(PSNetworkManagerCommit))]
     public class PostAzNetworkManagerCommitCommand : NetworkManagerBaseCmdlet
     {
+        // Breaking change 2022-05-01 release, plan to remove warning in 2022-07-01 release
+        public const string CommitTypeChangeDesc = "Commit types are now restricted to 'SecurityAdmin' and 'Connectivity'.";
```

Comment:
What is it used for?

Reply:
Whenever the user uses this command we will (or maybe just the first time when using this module version?), it will warn them of the breaking change.